Library/Formula/pangomm.rb

```diff
@@ -1,9 +1,7 @@
-require 'formula'
-
 class Pangomm < Formula
-  homepage 'http://www.pango.org/'
-  url 'http://ftp.gnome.org/pub/GNOME/sources/pangomm/2.34/pangomm-2.34.0.tar.xz'
-  sha256 '0e82bbff62f626692a00f3772d8b17169a1842b8cc54d5f2ddb1fec2cede9e41'
+  homepage "http://www.pango.org/"
+  url "http://ftp.gnome.org/pub/GNOME/sources/pangomm/2.36/pangomm-2.36.0.tar.xz"
```

Comment:
This one too:
https://download.gnome.org/sources/pangomm/2.36/pangomm-2.36.0.tar.xz